rsts/deployment/cluster_config/performance.rst

```diff
@@ -124,6 +122,8 @@ FlytePropeller configures a default version, which is better than the actual def
 
 .. note:: As you increase the number of workers in FlytePropeller it is important to increase the number of cpu's given to FlytePropeller pod.
 
+It is important to note that the kube API server tends to throttle requests transparently. This means that while tweaking performance by increasing the allowed frequency of kube API server requests (e.x. increasing FlytePropeller workers or relaxing kube client config rate limiting) there may be steep performance decreases for no apparent reason. In these situations, looking at kube API server request queue metrics can help indicate whether throttling is the culprit. Unfortunately, there is no one-size-fits-all solution here and a fair bit of experimenting is useful in configuring these parameters for your workload.
```

Comment:
```suggestion
It's worth noting that the Kube API server tends to throttle requests transparently. This means that while tweaking performance by increasing the allowed frequency of Kube API server requests (e.g., increasing FlytePropeller workers or relaxing Kube client config rate-limiting), there may be steep performance decreases for no apparent reason. Looking at the Kube API server request queue data in these cases can assist in identifying whether throttling is to blame. Unfortunately, there is no one-size-fits-all solution here, and customizing these parameters for your workload will require trial and error.
```

Reply:
I made the requested changes except for keeping the term "metrics" rather than switching to "data" as proposed. I think it's important to explicitly identify the emitted metrics from the request queue rather than being more generalized.